clang/test/SemaSYCL/array-kernel-param-neg.cpp

```diff
@@ -0,0 +1,44 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -fcxx-exceptions -verify -fsyntax-only %s
+
+// This test checks if compiler reports compilation error on an attempt to pass
+// a struct with non-trivially copyable type as SYCL kernel parameter.
```

Comment:
Array of non-trivially copyable structs?

Reply:
OK.